### DISCUSSION TITLE
Pre-recorded loops?

### DISCUSSION THREAD

**Discussion by justinjbaugh** [pschatzmann/arduino-audiokit#75] [Q&A] [answered]
Would it be possible to preload the loop files as WAVs?

I am working on an art installation project and need to play pre-recorded loops from triggers (mute and unmute tracks). I will not need the recording function. Just starting and stopping each track individually. 

Can the record function be disabled and still keep the track and playback control?

I'm an Arduino beginner, btw. Still learning the coding side.

**Comment by pschatzmann** [comment, marked as answer]
I am afraid you got the architecture all wrong! 

This project is just a Hardware Abstraction Layer (HAL) (=driver) to allow the [Arduino Audio Tools](https://github.com/pschatzmann/arduino-audio-tools) to use the full functionality of the ES7148, ES7210, ES7243, ES8311, ES8347, ES8388, TAS5805M, AC101 audio chips.

The Audio Tools is a powerful audio library which which you can program any type of audio functionality. If you are new to programming and Arduino however I do not recommend it as starting project and you must be ready to read the documentation provided in the Wiki and look at the class documentation...  

It is annoying to get questions for things that can be been answered by reading the documentation!

Finally the question: "Can the record function be disabled" does not make any sense: Recoding can be implemented by copying audio from an audio source to a storage audio sink. (E.g. memory or a SD), so if you don't implement this, there is no recording and thus there is nothing to be disabled.
